src/include/simeng/Core.hh

```diff
@@ -31,12 +66,16 @@ class Core {
   virtual const ArchitecturalRegisterFileSet& getArchitecturalRegisterFileSet()
       const = 0;
 
+  /** Send a syscall to the system's syscall handler. */
```

Comment:
rename `system's` to `SimOS's` as it could confusing which system the comment refers to.

Reply:
We can't have the syscall handler referencing the SimOS directly as it's not always sitting beneath it, e.g. with SST

Reply:
I've reworded for clarity

Reply:
I've reworded for clarity but haven't used SimOS directly as with something like SST, SimOS doesn't exist